### ISSUE TITLE
HM3D dataset

### ISSUE THREAD

**Issue by qwer137559** [bdaiinstitute/vlfm#58]
<img width="873" alt="d62672c8ced98fac7b3b5ef0178abc1" src="https://github.com/user-attachments/assets/c6ee5451-d816-4d7a-83ac-3f5e1179ba08">

What is the cause of this issue, is it due to an incomplete dataset?

**Comment by qwer137559** [comment]
i am use hm3d-val-habitat-v0.2.tar as val data

**Comment by qwer137559** [comment]
its from https://github.com/matterport/habitat-matterport-3dresearch

**Comment by jkl375** [comment]
Hello. I encountered the same error when running "python -m vlfm.run". Did you manage to solve it in the end?

**Comment by luohongk** [comment]
您好。我在运行“python -m vlfm.run”时也遇到了同样的错误。您最后解决了吗

**Comment by Leon12334** [comment]
> 您好。我在运行“python -m vlfm.run”时也遇到了同样的错误。您最后解决了吗
hi，can u solve your datasets' problem?

**Comment by luohongk** [comment]
> > 您好。我在运行“python -m vlfm.run”时也遇到了同样的错误。您最后解决了吗
> > hi，can u solve your datasets' problem?

yes,you need to change some data file directories, refer to https://github.com/bdaiinstitute/vlfm/issues/35#issuecomment-2817188141
